tests/mapping/swagger20_request_validator/required_validator_test.py

```diff
@@ -0,0 +1,23 @@
+import mock
+
+from jsonschema.exceptions import ValidationError
+
+from bravado.mapping.swagger20_validator import required_validator
+
+
+def test_fail_if_required_parameter_but_not_present():
+    param_schema = {'name': 'foo', 'in': 'query', 'required': True}
+    assert isinstance(required_validator(None, True, None, param_schema)[0],
+                      ValidationError)
+
+
+def test_pass_if_not_required_paramter_and_not_present():
+    param_schema = {'name': 'foo', 'in': 'query', 'required': True}
+    assert required_validator(None, False, None, param_schema) is None
```

Comment:
Fixed. Actually the value `required` was directly sent as the second parameter. Corrected now to pass the `required` field.